index.html

```diff
@@ -158,11 +177,47 @@
   scroll-snap-align: center;
 }
 
+lazy-youtube a {
+  display: grid;
+  justify-items: center;
+  width: 100%;
+  align-items: center;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+}
 
+lazy-youtube a:hover {
+  text-decoration: none;
+}
 
-#demos .btn { display: none; }
+lazy-youtube a:hover span {
+  background-color: white;
+}
 
+lazy-youtube a > * {
+  grid-row: 1;
+  grid-column: 1;
+  display: block;
+}
+lazy-youtube a > span {
+  z-index: 2;
+  width: 68px;
+  color: transparent;
+  height: 48px;
+  cursor: pointer;
+  transform: translate3d(-50%, -50%, 0);
+  margin-top: 68px;
+  margin-left: 68px;
+  z-index: 1;
+  background-color: transparent;
+  /* YT's actual play button svg */
+  background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 68 48"><path fill="%23f00" fill-opacity="0.8" d="M66.52,7.74c-0.78-2.93-2.49-5.41-5.42-6.19C55.79,.13,34,0,34,0S12.21,.13,6.9,1.55 C3.97,2.33,2.27,4.81,1.48,7.74C0.06,13.05,0,24,0,24s0.06,10.95,1.48,16.26c0.78,2.93,2.49,5.41,5.42,6.19 C12.21,47.87,34,48,34,48s21.79-0.13,27.1-1.55c2.93-0.78,4.64-3.26,5.42-6.19C67.94,34.95,68,24,68,24S67.94,13.05,66.52,7.74z"></path><path d="M 45,24 27,14 27,34" fill="%23fff"></path></svg>');
+  filter: grayscale(100%);
+  transition: filter .1s cubic-bezier(0, 0, 0.2, 1);
+  border: none;
+}
```

Comment:
I _think_ that adding something like the following might fix the white artifact on hover:

```css
lazy-youtube a:hover > span {
  background-color: transparent;
}
```

(Beware: completely untested!)